### PR TITLE
TH-1076: Show target age in event cards

### PR DESCRIPTION
**Pull request by melniiv**
## Additional notes :spiral_notepad:

### CODE REVIEW

**Comment by Sartonon** [comment]
We need to make sure how those ages should be written in english/swedish. They might be different compared to finnish

**Comment by melniiv** [comment]
it is. the same code is used in info, and its well translated

**Review by Sartonon — COMMENTED** [review]
Couple comments added. Maybe getAudienceMaxAge could use some refactoring

File: src/domain/event/EventUtils.ts [hunk 1]
```diff
@@ -386,3 +389,14 @@ export const isLocalized = (
       event.shortDescription?.[locale] &&
       event.description?.[locale]
   );
+
+export const getAudienceAge = (
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/event/EventUtils.ts]: fixed

File: src/domain/event/EventUtils.ts [hunk 2]
```diff
@@ -386,3 +389,14 @@ export const isLocalized = (
...
+  t: TFunction,
+  audienceMinAge: string | null | undefined,
+  audienceMaxAge: string | null | undefined
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/event/EventUtils.ts]: fixed

File: src/domain/event/EventUtils.ts [hunk 3]
```diff
@@ -386,3 +389,14 @@ export const isLocalized = (
...
+): string => {
+  const ageLimit = `${audienceMinAge ?? '0'}${
+    audienceMaxAge ? `-${audienceMaxAge}` : '+'
+  }`;
+  return ageLimit !== '0+' ? `${ageLimit} -${t('event.info.age')}` : '';
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/event/EventUtils.ts]: fixed

**Pull request merged**